packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -56,6 +56,13 @@ export interface ISemanticColors {
    */
   bodyBackground: string;
 
+  /**
+   * The standout color for highlighted content backgrounds.
+   * This should use the neutral variant when on no emphasis.
```

Comment:
More accurately,
"For highlighted content when there is no emphasis, use the neutral variant instead."